models/Challenge.js

```diff
@@ -0,0 +1,39 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Challenge extends Model {
+  //Create functions if needed
+}
+
+Challenge.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    starting_captial:{
```

Comment:
```suggestion
    starting_capital:{
```
Minor spelling error